app/controllers/application_templates_controller.rb

```diff
@@ -27,7 +27,7 @@ def toggle_active
     else show_message :inactive_application,
                       default: 'The application is now inactive.'
     end
-    redirect_to :back
+    redirect_back fallback_location: main_dashboard_path
```

Comment:
Is there any way to set this more globally? Seems pretty repetitive to write it everywhere.